src/test/resources/pages/empty-div.html

```diff
@@ -22,5 +22,14 @@
     }
 </script>
 
+<iframe id="myiframe" height="200" frameborder="0" title="description">
```

Comment:
Page is called `empty-div` and also the test name is `EmptyDivTestIT`. We probably shouldn't mix things up as this now also tests IFrames.

At least rename page and test I would suggest. We can also use different test methods/cases for different aspects.